### PR TITLE
#10 Use esn-frontend-inbox-calendar module to have calendar & resourc…

### PR DESCRIPTION
**Pull request by tuanlc**
…e blue-bar / indicator

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -78,8 +78,8 @@ module.exports = {
         '/calendar/app',
         '/linagora.esn.resource/api'
       ],
-      //target: 'http://localhost:8080',
-      target: 'https://dev.open-paas.org',
+      target: 'http://localhost:8080',
+      // target: 'https://dev.open-paas.org',
```
Comment (piavgh) [inline webpack.config.js:82]: We should not change this

Comment (tuanlc) [reply webpack.config.js]: This was a WIP to test on local. I reverted it

File: package.json [hunk 2]
```diff
@@ -16,7 +16,10 @@
     "angularjs-dragula": "2.0.0",
     "dompurify": "1.0.9",
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+    "esn-frontend-inbox-calendar": "github:openpaas-suite/esn-frontend-inbox-calendar#main",
+    "esn-frontend-mailto-handler": "github:openpaas-suite/esn-frontend-mailto-handler#main",
     "jmap-draft-client": "^0.1.2",
+    "fullcalendar": "3.9.0",
```
Comment (chamerling) [inline package.json:22]: DO we need to depend on it here?

File: src/index.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
+require('fullcalendar/dist/fullcalendar.js');
+require('fullcalendar/dist/locale-all.js');
```
Comment (chamerling) [inline src/index.js:2]: Why do we need this here? Should be in the module we already import, no?

File: package.json (hunk 2 above)
Comment (tuanlc) [reply package.json:22]: Moved this to the intermediary module

File: src/index.js (hunk 3 above)
Comment (tuanlc) [reply src/index.js]: Moved this to the intermediary module

File: package.json [hunk 4]
```diff
@@ -16,6 +16,8 @@
     "angularjs-dragula": "2.0.0",
     "dompurify": "1.0.9",
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+    "esn-frontend-inbox-calendar": "github:openpaas-suite/esn-frontend-inbox-calendar#main",
+    "esn-frontend-mailto-handler": "github:openpaas-suite/esn-frontend-mailto-handler#main",
```
Comment (chamerling) [inline package.json:20]: No, this one is used outside of inbox, we are in inbox, a mailto link will open the inbox composer, not the SPA

Comment (tuanlc) [reply package.json]: Reverted this

File: src/linagora.esn.unifiedinbox/views/thread/list/list-item.pug [hunk 5]
```diff
@@ -38,7 +38,9 @@
             .indicators.esn-flex-noshrink.visible-xl
               ul.actions.flex-vertical-centered
                 li.action-item
-                  i.inbox-message-indicators(dynamic-directive='inbox-message-indicators')
+                  i.inbox-message-indicators
+                    cal-inbox-invitation-message-indicator(ng-if="ctrl.shouldDisplayCalendarInvitationMessageIndicator")
+                    cal-inbox-resource-management-indicator(ng-if="ctrl.shouldDisplayCalendarResourceManagementIndicator")
```
Comment (huy-ta) [inline src/linagora.esn.unifiedinbox/views/thread/list/list-item.pug:43]: So when `ctrl.shouldDisplayCalendarInvitationMessageIndicator = false` and `ctrl.shouldDisplayCalendarResourceManagementIndicator = false`, only an empty `i.inbox-message-indicators` will be left right? Will it occupy some space and result in some weird UI errors?

Comment (tuanlc) [reply src/linagora.esn.unifiedinbox/views/thread/list/list-item.pug:43]: It works without UI errors before this PR

**Pull request merged**